content/en-US/api/fontKerning.md

```diff
@@ -0,0 +1,68 @@
+---
+recipe: api-interface
+title: 'CanvasRenderingContext2D.fontKerning'
+mdn_url: /en-US/docs/Web/API/CanvasRenderingContext2D/fontKerning
+specifications: [[specURL]]#[[HeadingAnchor]]
+browser_compatibility: api.CanvasRenderingContext2D.fontKerning
+---
+
+
+**When this feature ships, the content below will live on MDN under
+[developer.mozilla.org/en-US/docs/Web/CanvasRenderingContext2D/fontKerning](https://developer.mozilla.org/en-US/docs/Web/CanvasRenderingContext2D/fontKerning).**
+
+## Description
+
+The `fontKerning` property of the `CanvasRenderingContext2D` interface allows 
+the developers to set whether to use of the kerning information stored in a 
```

Comment:
"allows the developers to set whether to use of the kerning information stored" should be
"allows developers to set whether to use the kerning information stored"